### PR TITLE
Add trivia to TriviaService resolve #2 

### LINKED ISSUE #2: Add More Trivia
We need to add more trivia to [TriviaService](https://github.com/MoritzHayden/bosco/blob/main/src/service/trivia.py) for the `/trivia` command, as there are currently only two facts there. Perhaps we can scrape some trivia from the [Deep Rock Galactic Wiki](https://deeprockgalactic.fandom.com/wiki/Deep_Rock_Galactic_Wiki)?

### PR DESCRIPTION
**Pull request by hynrey**
## Summary

Added some trivia strings to TriviaService.
Added Mission Type trivia's, Equipment of classes

## Type of Change

Please select all that apply:

- [ ] Bug Fix
- [X] New Feature
- [ ] Documentation Update

### CODE REVIEW

File: src/service/trivia.py [hunk 1]
```diff
@@ -5,7 +5,50 @@ class TriviaService:
...
+            'The M1000 Classic is based on the American M1 Garand semi-automatic rifle, and includes the trademark "ping" when ejecting spent clips.',
+            'The DRAK-25 Plasma Carbine\'s weapon prototype used the Deepcore 40mm PGL model.',
+            'The Special Powder\'s effect of Jury-Rigged Boomstick is similar to Team Fortress 2\'s Force-a-Nature knockback jump, which is also used by the game\'s Scout class.',
+            'The "Double Trigger" upgrade of Jury-Rigged Boomstick not only increases your fire rate by 6x, it also decreases reload time by 0.55 seconds, making it almost always better than quickfire ejector.',
+            'The name of Zhukov NUK17 is likely a reference to Georgy Zhukov, a Soviet general during World War 2. The word "zhuk" or "жук" in russian translates to "bug", which could make sense in the context of the game.',
+            'Some weapon and upgrade descriptions reference R&D (which is short for Research and Development). This is the department of DRG responsible for creating new weapons and equipment for the Dwarves to use.'
```
Comment (MoritzHayden) [inline src/service/trivia.py:51]: The formatting looks good to me. I'm curious, where did you find these facts?

**Review by MoritzHayden — COMMENTED** [review]
Thank you for your contribution!

File: src/service/trivia.py (hunk 1 above)
Comment (hynrey) [reply src/service/trivia.py:51]: That all facts come from [DRG Wiki](https://deeprockgalactic.fandom.com/wiki/Deep_Rock_Galactic_Wiki)

Comment (MoritzHayden) [reply src/service/trivia.py:51]: Excellent, thanks!

**Review by MoritzHayden — APPROVED** [review]

**Pull request merged**